theme/ckeditor5-image/textalternativeform.css

```diff
@@ -3,25 +3,24 @@
  * For licensing, see LICENSE.md.
  */
 
-.cke-text-alternative-form {
-	padding: var(--ck-spacing-large);
+.ck-text-alternative-form {
+	padding: var(--ck-spacing-standard);
+
+	& .ck-button {
+		border-width: 0;
+	}
 
 	&:focus {
-		/* https://github.com/ckeditor/ckeditor5-image/issues/40 */
+		/* https://github.com/ckeditor/ckeditor5-link/issues/90 */
 		outline: none;
 	}
 
-	& .ck-label {
-		margin-bottom: var(--ck-spacing-small);
+	& > :not(:first-child) {
+		margin-left: var(--ck-spacing-standard);
 	}
-}
-
-.cke-text-alternative-form__actions {
-	padding-top: var(--ck-spacing-large);
 
-	& .ck-button {
-		& + .ck-button {
-			margin-right: var(--ck-spacing-medium);
-		}
+	& .ck-input-text {
+		/* https://github.com/ckeditor/ckeditor5-link/issues/145 */
+		min-width: 18em;
```

Comment:
Maybe we should use the variable like in 
```css
.ck-link-actions .ck-button.ck-link-actions__preview .ck-button__label { ... }
```
 🤔 ?

Reply:
Sounds like a plan.